### ISSUE TITLE
Required IAM permissions

### ISSUE THREAD

**Issue by dougtoppin** [jpsim/AWSPics#3]
You might consider including in the README the IAM role/policies that are required to complete the entire process. Since this has specific functionality it can be useful to do it using a discrete and limited set of capabilities in your account.

**Comment by jpsim** [comment]
I agree this is something that should be done. I likely won't get to it myself though, so if anyone wants to help out, PRs are appreciated!

**Comment by dougtoppin** [comment]
I started on it last night and will try a few combinations over the next week or two. Will PR if I can narrow it down. tks for the work and repo. I've been looking for something like this for a while.

**Comment by jpsim** [comment]
I'm glad to hear that. Thanks for your interest in contributing! 👏

**Comment by dougtoppin** [comment]
I started some work on this. The process that I am following is to create a IAM user with no perms and add them as needed for each function. I will also file a few issues as I go along and should be able to reference them when I make some progress.

**Comment by dougtoppin** [comment]
I expect to be able to put some time in on this again this weekend. More to come.
